Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/overlay/InputOverlay.java

```diff
@@ -569,14 +574,15 @@ private void saveControlPosition(int sharedPrefsId, int x, int y)
 	 * InputOverlayDrawableButton. Simply add it to the HashSet of overlay items and wait
 	 * for Android to call the onDraw method.
 	 * 
-	 * @param context  The current {@link Context}.
-	 * @param resId    The resource ID of the {@link Drawable} to get the {@link Bitmap} of.
-	 * @param buttonId Identifier for determining what type of button the initialized InputOverlayDrawableButton represents.
+	 * @param context  			The current {@link Context}.
+	 * @param defaultResId    	The resource ID of the {@link Drawable} to get the {@link Bitmap} of (Default State).
+	 * @param pressedResId    	The resource ID of the {@link Drawable} to get the {@link Bitmap} of (Pressed State).
+	 * @param buttonId 			Identifier for determining what type of button the initialized InputOverlayDrawableButton represents.
```

Comment:
In the C++ code, we use spaces only. In the Java code... well, I don't really think there is a rule at the moment. But you should at least not use tabs between these two columns, since it otherwise would be impossible to format this in a way that looks good regardless of the tab width setting.